ocaml/utils/clflags.ml

```diff
@@ -52,7 +52,6 @@ and debug = ref false                   (* -g *)
 and debug_full = ref false              (* For full DWARF support *)
 and unsafe = ref false                  (* -unsafe *)
 and use_linscan = ref false             (* -linscan *)
-and use_ocamlcfg = ref false            (* -ocamlcfg *)
 and link_everything = ref false         (* -linkall *)
```

Comment:
I think it would be worth doing a check (if you haven't already) on the final version of this PR by diffing this file against the 4.12 upstream version, just to make sure nothing has been missed.

Reply:
If you mean in comparison to ocaml/ subdirectory, I checked when I've move the module Clflags.Flambda2 and from compenv. Now if anyone adds any flags on `main` branch, this PR will be in conflict.